Samples/Interactivity/Commands/InteractiveCommands.cs

```diff
@@ -115,7 +118,7 @@ public async Task<IResult> SendDropdownAsync()
             {
                 new SelectMenuComponent
                 (
-                    "colour-dropdown",
+                    $"{Constants.InteractionTree}::{Constants.SelectMenuPrefix}::colour-dropdown",
                     new ISelectOption[]
                     {
                         new SelectOption("Red", "#FF0000"),
```

Comment:
If an buider rout is added the SelectMenuBuilder could have an ``AddSelectOption`` method which will add the select options to an internal list that the ``Build()`` method would ``.ToArray()`` on when creating the ``SelectMenuComponent``.